qiskit/utils/quantum_instance.py

```diff
@@ -304,6 +305,23 @@ def execute(self,
         if not had_transpiled:
             circuits = self.transpile(circuits)
 
+        # fix for providers that don't support QasmQobj until a bigger refactor happens
+        from qiskit.providers import BackendV1
+        if isinstance(self._backend, BackendV1) and \
+                not is_aer_provider(self._backend) and \
+                not is_basicaer_provider(self._backend) and \
+                not is_ibmq_provider(self._backend):
+            result = run_circuits(circuits,
+                                  self._backend,
+                                  self.backend_options,
+                                  self.qjob_config,
+                                  self.run_config,
+                                  self._job_callback)
+            self._time_taken += result.time_taken
+            if self._circuit_summary:
+                self._circuit_summary = False
+            return result
```

Comment:
So this means that measurement mitigation is not an option here right? Is that a tradeoff we're ok with? Nothing in ignis should require qobj so this should just be a matter of getting the physical qubits from the transpiled circuits to figure out which bits to build a fitter (and to append the cal circuits to the job if needed).

Reply:
Unless we figured out a way to rewrite the error mitigation logic in QuantumInstance and measurement_error_mitigation.py to eliminate QObj, we can't use Aer without Qobj.

Reply:
The mitigation logic should be straightforward to do with circuits it's basically identical steps but you don't build a qobj just pass the calibration circuit directly to the backend. The actual fitter should work the same with the result object once you have the cal matrix. The only functional difference will be how you get the physical qubits from the circuit, but this isn't hard either, and it looks like there is already a function for it: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/utils/measurement_error_mitigation.py#L21

Reply:
@manoelmarques and I did this together in https://github.com/Qiskit/qiskit-terra/pull/6299/commits/36e4dcc07eb3027d3642fad312e441a1a3fa7367